crates/uv-torch/src/accelerator.rs

```diff
@@ -150,6 +167,39 @@ impl Accelerator {
             }
         }
 
+        // Read from `/sys/bus/pci/devices` to filter for Intel GPU via PCI.
+        match WalkDir::new("/sys/bus/pci/devices")
+            .min_depth(1)
+            .max_depth(1)
+            .into_iter()
+            .collect::<Result<Vec<_>, _>>()
+        {
```

Comment:
Is this different than using `fs::read_dir`, since we're only going one depth?

Reply:
(Apart from this, the rest of the change looks good and I'm happy to merge once resolved.)

Reply:
CI clippy told me `fs::read_dir` is disallowed in `uv`, please refer to https://github.com/astral-sh/uv/blob/5e2dc5a9aa18aaa942f26513ca3b8ae4704eb018/clippy.toml#L21-L29

Reply:
Ohh, it wants you to use `fs_err::read_dir`.

Reply:
@charliermarsh Thanks for the suggestion! I’ve switched to using `fs_err::read_dir` as recommended. Both the CI and local checks now pass.
![image](https://github.com/user-attachments/assets/c5c05406-b2af-498d-ab88-028606a0a014)